events.go

```diff
@@ -53,22 +56,38 @@ func main() {
 
 	if len(serverConfig.AMQP1EventURL) == 0 {
 		log.Println("AMQP1 Event URL is required")
-		usage()
+		eventusage()
 		os.Exit(1)
 	}
 	if len(serverConfig.ElasticHostURL) == 0 {
 		log.Println("Elastic Host URL is required")
-		usage()
+		eventusage()
 		os.Exit(1)
 	}
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, os.Interrupt)
+	go func() {
+		for sig := range c {
+			// sig is a ^C, handle it
+			log.Printf("caught sig: %+v", sig)
+			log.Println("Wait for 2 second to finish processing")
+			time.Sleep(2 * time.Second)
+			os.Exit(0)
+		}
+	}()
 
+	log.Printf("Config %#v\n", serverConfig)
 	eventsNotifier := make(chan string) // Channel for messages from goroutines to main()
 	var amqpEventServer *amqplistener.AMQPServer
 	///Metric Listener
 	amqpEventsurl := fmt.Sprintf("amqp://%s", serverConfig.AMQP1EventURL)
+	log.Printf("Connecting to AMQP1 : %s\n", amqpEventsurl)
 	amqpEventServer = amqplistener.NewAMQPServer(amqpEventsurl, true, -1, eventsNotifier)
+	log.Printf("Listening.....\n")
 	var elasticClient *saelastic.ElasticClient
+	log.Printf("Connecting to Elastic Search : %s\n", serverConfig.ElasticHostURL)
```

Comment:
Just to be pedantic, it should be "ElasticSearch"

Reply:
Fixed